pyaedt/generic/filesystem.py

```diff
@@ -97,8 +103,8 @@ def copyfile(self, src_file, dst_filename=None):
                 pass
         try:
             shutil.copy2(src_file, dst_file)
-        except:  # pragma: no cover
-            pass
+        except FileNotFoundError as fnf_error:
+            print(fnf_error)
```

Comment:
Why not using the log here? Is is because this function can be call before the log has been instantiated?

Reply:
@MaxJPRey: I will add a log statement.

Reply:
This was an older change, so I need a bit more time to review and add a unit test. I will add an issue and fix it later.